[CI] CCSFieldCapabilitiesIT.testFailuresFromRemote

**Build scan**:

https://gradle-enterprise.elastic.co/s/mvr6xm42r4txw/tests/:server:internalClusterTest/org.elasticsearch.search.fieldcaps.CCSFieldCapabilitiesIT/testFailuresFromRemote#1

**Repro line**:

```
./gradlew ':server:internalClusterTest' --tests "org.elasticsearch.search.fieldcaps.CCSFieldCapabilitiesIT.testFailuresFromRemote" -Dtests.seed=6B25B2499A82C5 -Dtests.security.manager=true -Dtests.locale=de-GR -Dtests.timezone=Asia/Ho_Chi_Minh -Druntime.java=8
```

**Reproduces locally?**: Yes

**Applicable branches**: master, 7.x

**Failure excerpt**:
```
java.lang.AssertionError: expected:<class java.lang.IllegalArgumentException> but was:<class org.elasticsearch.index.shard.IllegalIndexShardStateException>
        at __randomizedtesting.SeedInfo.seed([6B25B2499A82C5:627B2742E112BAF1]:0)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at org.elasticsearch.search.fieldcaps.CCSFieldCapabilitiesIT.testFailuresFromRemote(CCSFieldCapabilitiesIT.java:75)
```

--- a/server/src/internalClusterTest/java/org/elasticsearch/search/fieldcaps/CCSFieldCapabilitiesIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/search/fieldcaps/CCSFieldCapabilitiesIT.java
@@ -12,6 +12,7 @@ import org.elasticsearch.ExceptionsHelper;
 import org.elasticsearch.action.fieldcaps.FieldCapabilitiesFailure;
 import org.elasticsearch.action.fieldcaps.FieldCapabilitiesResponse;
 import org.elasticsearch.client.Client;
+import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.plugins.Plugin;
 import org.elasticsearch.search.fieldcaps.FieldCapabilitiesIT.ExceptionOnRewriteQueryBuilder;
 import org.elasticsearch.search.fieldcaps.FieldCapabilitiesIT.ExceptionOnRewriteQueryPlugin;
@@ -41,17 +42,17 @@ public class CCSFieldCapabilitiesIT extends AbstractMultiClustersTestCase {
         return plugins;
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/71349")
     public void testFailuresFromRemote() {
+        Settings indexSettings = Settings.builder().put("index.number_of_replicas", 0).build();
         final Client localClient = client(LOCAL_CLUSTER);
         final Client remoteClient = client("remote_cluster");
         String localIndex = "local_test";
-        assertAcked(localClient.admin().indices().prepareCreate(localIndex));
+        assertAcked(localClient.admin().indices().prepareCreate(localIndex).setSettings(indexSettings));
         localClient.prepareIndex(localIndex).setId("1").setSource("foo", "bar").get();
         localClient.admin().indices().prepareRefresh(localIndex).get();
 
         String remoteErrorIndex = "remote_test_error";
-        assertAcked(remoteClient.admin().indices().prepareCreate(remoteErrorIndex));
+        assertAcked(remoteClient.admin().indices().prepareCreate(remoteErrorIndex).setSettings(indexSettings));
         remoteClient.prepareIndex(remoteErrorIndex).setId("2").setSource("foo", "bar").get();
         remoteClient.admin().indices().prepareRefresh(remoteErrorIndex).get();
 

